CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing
+
+Thank you for considering to contribute to Surfbird, whether it is a simple bug report or an entire feature you might have planned out, everything is appreciated and welcome!
+
+### I have a problem/question!
+
+Do you have a question regarding Surfbird or a specific problem with something? if there is no issue for it yet, [please open one](https://github.com/surfbirdapp/surfbird/issues/new)!
+
+### I found a bug!
+
+If you think you found a bug in Surfbird, please search through [all issues](https://github.com/surfbirdapp/surfbird/issues?q=) before opening another one!
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/surfbirdapp/surfbird/issues/new). Be sure to include a title and clear description, as much relevant information as possible, maybe even include a screenshot, the more the better!
+
+### I think I can fix/add it!
+
+Awesome! You can follow our [Development Setup](docs/development-setup.md) guide to set up a development environment for Surfbird!
+
+Once the setup is up and running, you're ready to implement your feature or fix.
+
+* [Fork](https://github.com/surfbirdapp/surfbird/fork) the repository
+* Create a branch for your changes
+  * Use Git Flow-like `feature/` or `fix/` prefixes for your branch names
```

Comment:
I'd link to some good [Git Flow documentation](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow), in case people haven't heard of it before and want to read into it 👌